### ISSUE TITLE
'참고'와 '참조'

### ISSUE THREAD

**Issue by HuidaeCho** [HuidaeCho/grass-ko#7]
[참고와 참조의 차이](https://korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=57239)

'이런저런 문서를 참고하세요.'

**Comment by yungjinsung** [comment]
말씀하신대로, "참고"와 "참조"의 의미로 보자면 "참고"가 맞습니다. 하지만, 국내에서 많은 문서들이 "참조"를 쓰고 있어서, 어쩔 수 없는 부분들이 있네요.

**Comment by HuidaeCho** [comment]
맞습니다. 거기에 대한 [나무위키 문서](https://namu.wiki/w/%EC%B0%B8%EA%B3%A0%EC%99%80%20%EC%B0%B8%EC%A1%B0)도 있습니다.

> 다만 요즘은 '참조'의 의미가 좀 더 광범위하게 쓰이는 방향으로 언어의 의미가 변하고 있는 상황이다. 서로 비교하는 것 자체가 목적인 경우에만 쓰이지 않고, '참고'를 써야할 상황에서도 '참조'를 써야 하는 식으로 '참고'의 영역을 잠식하는 상황이다. 아마도 학계에서 reference가 참조로 번역되어 사용된 것이 상당한 영향을 끼친 것으로 보인다.

그래서 나름대로 'reference system'은 관행대로 '참조 체계'로 쓰고 'refer to something'은 '참고'로 하는 것이 좋다고 생각했습니다.
